src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -19,8 +20,14 @@ public void actionPerformed(ActionEvent e) {
 		if(e.getActionCommand().equals("Figure Joe 1")) {
 			FiguresJoe.figureScript1(driverManager.getCurrentDriver());
 		}
-		else{
+		else if (e.getActionCommand().equals("Figure Joe 2")){
 			FiguresJoe.figureScript2(driverManager.getCurrentDriver());
 		}
+		else if (e.getActionCommand().equals("Square")){
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.